deviceaccess.py

```diff
@@ -1206,16 +1206,21 @@ def getRegisterCatalogue(self) -> pb.RegisterCatalogue:
         """
         return self._device.getRegisterCatalogue()
 
-    def read(self, registerPath: str, dtype: np.dtype = np.float64, wordOffsetInRegister: int = 0,
-             accessModeFlags: Sequence[AccessMode] = None) -> np.ndarray | np.number:
+    def read(self, registerPath: str, dtype: np.dtype = np.float64, numberOfWords: int = 0, numberOfChannels: int = 0, wordOffsetInRegister: int = 0, accessModeFlags: Sequence[AccessMode] = None) -> np.ndarray | np.number:
```

Comment:
isn't that line longer than 120 characters?

Reply:
It's 119.

Reply:
no, it's longer, sorry. without the indentation it is 223 characters.

Reply:
just to be sure: I refer to the line as it is shown by github, which claims you merged the old lines 1209 and 1210 into the single line 1209 (and added a parameter which makes it even longer). The original lines seem to be formatted properly, at least they were < 120 chars but > 80 chars.